lib/bios/menus-dasharo.robot

```diff
@@ -559,16 +376,24 @@ Save Changes
     Sleep    2s
 
 Save Changes And Reset
-    [Documentation]    Saves current UEFI settings and restarts.
+    [Documentation]    Saves current UEFI settings and restarts. ${nesting_level}
+    ...    is how deep user is currently in the settings.
+    ...    ${main_menu_steps_to_reset} means how many times should
+    ...    arrow down be pressed to get to the Reset option in main
+    ...    settings menu
+    # robocop: disable=unused-argument
```

Comment:
@m-iwanicki can you help there? I think you added this change (it was in my commit by rebasing and squashing some of them).

Reply:
@TomaszAIR @macpijan I think I changed how reset was done so it could be done without knowing/remembering how to get to Reset option from where we currently are.
I left those default unused arguments because there are other places where this keyword is used with them and if I remember correctly there were other PR's that used them that would stop working when merged.